src/configure/configurers/remoteGitHubWorkflowConfigurer.ts

```diff
@@ -47,15 +48,15 @@ export class RemoteGitHubWorkflowConfigurer extends LocalGitHubWorkflowConfigure
     public async getInputs(wizardInputs: WizardInputs): Promise<void> {
         this.inputs = wizardInputs;
         this.githubClient = new GithubClient(wizardInputs.githubPATToken, wizardInputs.sourceRepository.remoteUrl);
-        this.templateServiceClient = new TemplateServiceClient(wizardInputs.azureSession.credentials);
+        this.templateServiceClient = await TemplateServiceClientFactory.getClient(wizardInputs.azureSession.credentials);
```

Comment:
shouldn't we be passing pat token as well so that it can be used in case of moda client